src/test/run-make/pretty-expanded-hygiene/input.rs

```diff
@@ -16,6 +16,12 @@ macro_rules! foo {
     ($x: ident) => { y + $x }
 }
 
+#[derive(Copy, Clone)]
+struct A { a: i32 }
+
+#[derive(Copy, Clone)]
+struct B<T> { a: T }
+
```

Comment:
Can you put this in its own rmake test case instead of piggy-backing on this one?